core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/clustering/hazelcast/aws/AWSBasedMembershipScheme.java

```diff
@@ -119,42 +118,42 @@ public void init() throws ClusteringFault {
             if (secretResolver != null) {
                 String resolvedValue = MiscellaneousUtil.resolve(accessKey.getParameterElement(), secretResolver);
                 if (!StringUtils.isEmpty(resolvedValue)) {
-                    awsConfig.setAccessKey(resolvedValue);
+                    awsConfig.setProperty("access-key", resolvedValue);
```

Comment:
Why can't we reuse the existing method here?
Check for other properties as well

Reply:
The existing methods were deprecated. For the 4.x migration guide I could see the properties were set as above